Source/Extensions/Blazorise.Components/Autocomplete.razor

```diff
@@ -11,7 +11,17 @@
         {
             @foreach ( var selectedText in SelectedTexts )
             {
-                <Badge Color="@MultipleBadgeColor" Margin="Margin.Is2.FromStart" CloseClicked="async () => await RemoveMultipleTextAndValue( selectedText )">@selectedText</Badge>
+                @if ( BadgeTemplate is null )
+                {
+                    <Badge Color="@MultipleBadgeColor" Margin="Margin.Is2.FromStart" CloseClicked="async () => await RemoveMultipleTextAndValue( selectedText )">@selectedText</Badge>
+                }
+                else
+                {
+                    var item = GetItemByText( selectedText );
+                    var value = GetItemValue( selectedText );
+                    var removeCallback = EventCallback.Factory.Create( this, async () => await RemoveMultipleTextAndValue( selectedText ) );
```

Comment:
Is there any performance hits for doing `EventCallback.Factory.Create` on each render?

Reply:
Well, same hit as getting the item and value. It has to be recalculated if it goes through the loop and collection hasn't changed.

It is of note that `Eventcallback` is a struct, the biggest hit is from the anonymous delegate.

Only workaround I know is this : 
(https://learn.microsoft.com/en-us/aspnet/core/blazor/performance?view=aspnetcore-7.0#avoid-recreating-delegates-for-many-repeated-elements-or-components)
![image](https://github.com/Megabit/Blazorise/assets/22283161/c6bdca8b-03ef-4843-8db7-97d99226e26d)
But since the collection is dynamic we would still need to recalculate each time and to make it efficient try to do it diff based, So I'm not sure there would be a significant benefit in relation to the added complexity, `Autocomplete` is already huge, eh.

---

Reply:
Ok. it can stay in that case. Thank you for the clarification.